### PR TITLE
Update to version 0.0.8, update recipe

### PR DESCRIPTION
**Pull request by dbast**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dbast** [comment]
Maybe this is also noarch: generic

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ package:
 
 source:
   url: https://github.com/iobis/obistools/archive/v{{ version }}.tar.gz
-  sh256: ea7656717d36e936c2c64810e13f1c59cfaef5ba5feb367b2dc2e42828541029
+  sha256: ea7656717d36e936c2c64810e13f1c59cfaef5ba5feb367b2dc2e42828541029
```
Comment (ocefpaf) [inline recipe/meta.yaml:11]: oops! Copy-n-pasta!

**Comment by dbast** [comment]
Interesting recipe failure: Solving environment: ...working... Too long with no output (exceeded 10m0s)

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by dbast** [comment]
The failure seems to be related to r-sf (from defaults, not migrated yet) and r-rgeos (or geos respectively). I can see the same failure when trying to build an environment locally with all the dependencies, but succeed when excluding those two packages.

**Comment by dbast** [comment]
@conda-forge-admin, please rerender

**Comment by dbast** [comment]
@ocefpaf Ready to merge :)

**Pull request merged**